cohort-26-website-committee/class-website
# Icon Slider

# Description
This feature implements the icon slider functionality. Will need a switch out the png file with new png file that includes all the technologies described on ticket #4  
## Type of change
Changes were made to agency.css for styling and animation. Keyframes may require another `npm i`. 
Added two divs to index.html.
Package.json was updated for keyframes dependency.
Created _imagescroll.scss for sass
I accidentally deleted _masthead.scss and committed. I have recreated the file.
Added line in agency.scss to import _imagescroll.scss
# Testing Instructions
`git fetch --all` and checkout the branch `git checkout ss-Slider`
in the terminal inside project directory class-website, do `http-server -o`
the web browser should run the current build with the addition of the icon slider below class picture

# Checklist:
